### ISSUE TITLE
The functionality for round navigation of OpenGotha tournaments is broken

### ISSUE THREAD

**Issue by juanchodepisa** [online-go/online-go.com#2041]
**Describe the bug**
Following on [this thread](https://forums.online-go.com/t/what-happened-to-gotha-tournaments-i-seems-thoroughly-broken/45135/7).
When creating a new OpenGotha tournament (or navigating old ones) the buttons for "Roster", "Standings" and round navigation are not working properly.
To wit round navigation seems to be entirely non-functional, whereas Roster and Standings show instead the pairings for the first round (if any).
The problem seems to be in the tournament page and not on the database, because the API properly shows registered players, and indicate that the rounds and pairings do exist.

**To Reproduce**
1. Create an _OpenGotha_ tournament with multiple rounds.
2. Add players to the tournament (by any method, this seems to be working well).
3. Download xml the file.
4. Add several games on different rounds (using OpenGotha) and save.
5. Reupload the xml file (dialog indicates the games are added correctly).
6. Click "start tournament" (the tournament starts).
7. Try navigating the rounds on the tournament page (nothing shows up).
9. Click start round. The following dialog appears: "Start round [object Object]1 now?", "Cancel", "Ok"
10. Click "Ok". An error message is shown: "Not found the requested resource".
11. Click on "Roster". It shows the pairings for the first round. The "Start Round" button is also available.
12. Click "Start Round" and "Ok". The first round is started.

**Expected behavior**
Navigating rounds should show the pairings for each round. The "Start Round" button should start the current round (instead of showing an error).
The "Roster" button should show the players, not the pairings. Also, it should not have "Start Round" as an available option.

**Comment by anoek** [comment]
Can you attach or email me (anoek@online-go.com) the xml file you're trying to use?

**Comment by juanchodepisa** [comment]
Sorry, I hadn't had time to follow up on this.

A few things:

* I do believe the problem does not lie in the xml file being uploaded (all of that works just fine). As "evidence" of this belief I'll point out that *old tournaments* now appear bugged, when they previously didn't. For example, this one: https://online-go.com/tournament/80176
* I suspect (have a hunch) that this change had something to do with it: https://github.com/online-go/online-go.com/pull/2007/files#diff-60899bcbb55f21a4bf440f7ebf1dca32d503d97b7909642f41ead5705de0dc4b in case that helps narrowing it down.
* I went ahead and created a new test tournament https://beta.online-go.com/tournament/175 on the beta website. It's a 3 round tournament which has games scheduled for all 3 rounds. 

I confirm that everything works correctly up to the point when you "Close Registration". From that point on, the bug appears.
The games are not started yet, but they were accepted by the system (I got the confirmation message). In fact, I cannot start them because the interface won't let me.

However, I **could** start the first round if I go to "Roster" and click on "Start Round". That is a case of two wrongs cancelling each other out and making a right.

Anyway, I have sent you the xml file of this tournament to your email, as you requested.
